### ISSUE TITLE
ImGuiFreeTypeBuilderFlags_LoadColor crash

### ISSUE THREAD

**Issue by N1ceL** [ocornut/imgui#4202]
Hello, I'm using ImGui with Shadows v1.83 + FreeType v2.10.4 and when trying to use the new flag ImGuiFreeTypeBuilderFlags_LoadColor I see this error:

![image](https://user-images.githubusercontent.com/41979846/120909019-c0bd2900-c6b3-11eb-9202-c7b5721e2644.png)

Using this code to load font:
```    static ImWchar ranges[] = { 0x1, 0x1FFFF, 0 };
    static ImFontConfig cfg;
    cfg.OversampleH = cfg.OversampleV = 1;
    cfg.MergeMode = true;
    cfg.FontBuilderFlags |= ImGuiFreeTypeBuilderFlags_LoadColor;
    io.Fonts->AddFontFromFileTTF("C:\\Windows\\Fonts\\seguiemj.ttf", 16.0f, &cfg, ranges);
```

**Comment by N1ceL** [comment]
So, with commented function ImFontAtlasBuildRenderShadowTexData all works

**Comment by ocornut** [comment]
Hello,
We cannot answer your question if you don’t provide the information requested by issue templatez

**Comment by rollraw** [comment]
> So, with commented function ImFontAtlasBuildRenderShadowTexData all works, but thats not 1 problem, emojis with white backgroud and looks that emojis without anti-aliasing

just in case try to build texdata without DE_MULTIPLY define so just r, g, b, a
https://github.com/ocornut/imgui/blob/fe245914114588f272b0924538fdd43f6c127a26/misc/freetype/imgui_freetype.cpp#L321-L331

**Comment by ocornut** [comment]
I pushed a fix in that branch for the first issue mentioned (the crash):
9c1ba7e82e81e89fd68a745f7a96f5d98573504f

For the second issue
> emojis with white backgroud and looks that emojis without anti-aliasing

I cannot repro and would need more details (which backend + repro).

**Comment by ocornut** [comment]
@N1ceL any update on this?

>> So, with commented function ImFontAtlasBuildRenderShadowTexData all works, but thats not 1 problem, emojis with white backgroud and looks that emojis without anti-aliasing
>  cannot repro and would need more details (which backend + repro).

**Comment by ocornut** [comment]
@N1ceL Closing this in absence of a reaction, as the primary issue reported was fixed already.
If you still have the "white background" problem please open an issue will all relevant details.

**Comment by 0x4d696e68** [comment]
maybe too later but this fix white background problem, imgui 1.88 + opengl2 backend

> > So, with commented function ImFontAtlasBuildRenderShadowTexData all works, but thats not 1 problem, emojis with white backgroud and looks that emojis without anti-aliasing
> 
> just in case try to build texdata without DE_MULTIPLY define so just r, g, b, a
> 
> https://github.com/ocornut/imgui/blob/fe245914114588f272b0924538fdd43f6c127a26/misc/freetype/imgui_freetype.cpp#L321-L331
